context_builder.Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Builds baseten/truss-context-builder, a light-weight image that can be used
+# for creating docker build context out of a Truss.
+# Build that image as:
+# docker buildx build . -f context_builder.Dockerfile --platform=linux/amd64 -t baseten/truss-context-builder
+FROM python:3.9-alpine
```

Comment:
Does it matter that this python version is fixed, but that the truss version can be changed based on `config.yaml`?

Reply:
Good catch. In practice where does this cause a problem? If someone uses python >3.10 functionality?

Reply:
someone can set pyversion to 3.7 or 3.8 and they would not get the python version that they expect. I guess I'm not sure if this matter because the base docker image for the server/training job itself didn't change and still respects the python version

Reply:
This version should not interfere with the version in truss config. This image is used to run truss library to generate the docker build context from a given truss. This transformation does not depend on python version where the truss library is used.